### PR TITLE
feat(Text): Add new 12px font size (Text3)

### PR DESCRIPTION
**Pull request by shaharzil**
https://monday.monday.com/boards/3532714909/pulses/6418938469

### CODE REVIEW

File: packages/core/src/components/Text/__stories__/Text.stories.js [hunk 1]
```diff
@@ -35,29 +35,57 @@ export const Overview = {
...
+          This is text1 bold
+        </Text>
+        <Text type={Text.types.TEXT1} weight={Text.weights.MEDIUM}>
+          This is text1 medium
+        </Text>
+        <Text type={Text.types.TEXT1} weight={Text.weights.NORMAL}>
+          This is text1 normal
+        </Text>
+      </Flex>
+      {/* todo - add spaces and dos and donts */}
```
Comment (talkor) [inline packages/core/src/components/Text/__stories__/Text.stories.js:59]: Comment

File: packages/core/src/components/Text/__stories__/Text.stories.js [hunk 2]
```diff
@@ -35,29 +35,57 @@ export const Overview = {
...
+          This is text2 bold
+        </Text>
+        <Text type={Text.types.TEXT2} weight={Text.weights.MEDIUM}>
+          This is text2 medium
+        </Text>
+        <Text type={Text.types.TEXT2} weight={Text.weights.NORMAL}>
+          This is text2 normal
+        </Text>
+      </Flex>
+      <Flex
```
Comment (talkor) [inline packages/core/src/components/Text/__stories__/Text.stories.js:76]: We said that we're gonna support bold but not document it, right?

**Review by talkor — APPROVED** [review]
Nice!

File: packages/core/src/components/Text/__stories__/Text.stories.js (hunk 2 above)
Comment (YossiSaadi) [reply packages/core/src/components/Text/__stories__/Text.stories.js:75]: Don't we want to disable it typescript-wise?
and to not allow the mixin variants for it?

Comment (shaharzil) [reply packages/core/src/components/Text/__stories__/Text.stories.js:75]: No, we don't want to block them so it won't look like a bug. we will just guide them to use It right (in Figma)

Comment (YossiSaadi) [reply packages/core/src/components/Text/__stories__/Text.stories.js:75]: got ya 👍🏼

**Pull request merged**